release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -277,6 +287,12 @@ For more information, see xref:../authentication/managing_cloud_provider_credent
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
```

Comment:
Just checking - did you decide against adding a separate "etcd" section under the new features section?